core/src/packet.rs

```diff
@@ -335,7 +335,9 @@ pub fn to_blob<T: Serialize>(resp: T, rsp_addr: SocketAddr) -> Result<Blob> {
     let mut b = Blob::default();
     let v = bincode::serialize(&resp)?;
     let len = v.len();
-    assert!(len <= BLOB_SIZE);
+    if len <= BLOB_SIZE {
```

Comment:
lol I just spent the last 15mins wondering why _nothing_ was working hehe

Reply:
oh, lol :)